stackapi/stackapi.py

```diff
@@ -68,7 +70,7 @@ def __init__(self, name=None, version="2.2", **kwargs):
         self._version = version
         self._previous_call = None
 
-        self._base_url = 'https://api.stackexchange.com/{}/'.format(version)
+        self._base_url = '{}/{}/'.format(base_url, version)
         sites = self.fetch('sites', filter='!*L1*AY-85YllAr2)', pagesize=1000)
```

Comment:
I'll need to test this to make sure the filter also exists on the enterprise version

Reply:
Can this be confirmed before merging?

Reply:
Yes, I should have time to try it out today

Reply:
@AWegnerGitHub Confirmed that the filter does exist ✅